config/initializers/new_framework_defaults_5_1.rb

```diff
@@ -1,14 +0,0 @@
-# Be sure to restart your server when you modify this file.
-#
-# This file contains migration options to ease your Rails 5.1 upgrade.
-#
-# Once upgraded flip defaults one by one to migrate to the new default.
```

Comment:
This file was, I believe, redundant, as we were already using the Rails 5.1 defaults, via the `config.load_defaults 5.1` statement in `application.rb`.